modules/37_industry/subsectors/datainput.gms

```diff
@@ -550,6 +602,24 @@ execute_load "input_ref.gdx", vm_demFeSector_afterTax;
     );
 );
 
+*** maximum share of biomass use in industry subsector
+*** set to prevent too fast or technically unrealistic switch to biomass in industry subsectors
+
+*** initialize maximum biomass share in all industry subsectors at 100%
+p37_BioShareMaxSubsec(t,regi,entyFeCC37,secInd37)=1;
+
+
+*** for steel to prevent too fast switch to biomass, set maximum biomass shares in near-term
+p37_BioShareMaxSubsec(t,regi,"fesos","steel")$(t.val gt 2015 AND t.val le 2025)=0.3;
+p37_BioShareMaxSubsec("2030",regi,"fesos","steel")=0.4;
+p37_BioShareMaxSubsec("2035",regi,"fesos","steel")=0.7;
+p37_BioShareMaxSubsec("2040",regi,"fesos","steel")=1;
```

Comment:
This will be infeasible at least once @Renato-Rodrigues makes a dedicated Brazil region.  See [this discussion](https://mattermost.pik-potsdam.de/rd3/pl/7b5wzb66gf8t7kj1rtbcg8s35a).